noir-projects/noir-protocol-circuits/crates/types/src/abis/accumulated_data/combined_accumulated_data.nr

```diff
@@ -31,12 +32,15 @@ struct CombinedAccumulatedData {
 
 impl CombinedAccumulatedData {
     pub fn recombine(non_revertible: PublicAccumulatedData, revertible: PublicAccumulatedData) -> Self {
+        // TODO(Miranda): Hash here or elsewhere?
+        let encrypted_logs_hash = compute_tx_logs_hash(revertible.encrypted_logs_hashes);
+        let unencrypted_logs_hash = compute_tx_logs_hash(revertible.unencrypted_logs_hashes);
```

Comment:
The final single logs hash is calculated here as it has the lowest code impact. But I'm happy to move it since this is more of a conversion/struct file.